Xamarin.Forms.Platform.Android/Renderers/FormsVideoView.cs

```diff
@@ -39,7 +39,11 @@ void ExtractMetadata(MediaMetadataRetriever retriever)
 			if (!string.IsNullOrEmpty(durationString))
 			{
 				long durationMS = long.Parse(durationString);
```

Comment:
`durationString` will always be correctly parsed? Perhaps use `TryParse` as above.